api/base/serializers.py

```diff
@@ -266,6 +266,7 @@ def __init__(self, related_view=None, related_view_kwargs=None, self_view=None,
         self.related_meta = related_meta
         self.self_meta = self_meta
         self.always_embed = always_embed
+        self.source_field = source_field
```

Comment:
I see. The `source` attribute defines how values are pulled off of the objects being serialized. It'd be best not to mess with that.

However, "source_field" is a bit of a misnomer; it implies that it, too, has something to do with pulling values from the objects to serialize. In reality, it defines the key used for filtering the field. Maybe rename to "filter_key"?